### ISSUE TITLE
symbols that correspond to the names of predefined aspis elements resolve incorrectly

### ISSUE THREAD

**Issue by jumblerg** [HostelRocket/aspis#1]
if a newly-defined symbol collides with the name of a preexisting aspis/react/html element (as the menu element below does), the symbol will be resolved to the aspis element constructor instead of following clojure's scoping rules.

since the most commonly used terms in web development are already occupied by html elements (aspis reserves 128 of them), i encounter this problem frequently in the form of unexpected runtime behavior.

i've experienced this problem with both local and namespace-level bindings.

``` clojure
(defelem menu
  (div :css {:backgroundColor "#161B25"}
    (div :class "container"
      (div :class "row"
        (div :class "col-md-12"
          (ul :class "nav nav-dark-pills nav-dark-justified" 
            this/props.children ))))))

(defelem 
  (menu (button "hello")) ;; will subtly resolve to html menu, causing elements of custom menu to be ignored
```

**Comment by brendanyounger** [comment]
I agree this is a bit confusing.  Hoplon supported this by essentially inserting a [:require [tailrecursion.hoplon :refer [a b button...]] at the top of every page.  Since aspis isn't also in charge of compiling the files, it can't do that.

The alternative would be to use the namespaced name `a/div`, etc. for all references to html entities.  I originally did that, but thought the benefits of having `(div )` instead of `(a/div )` everywhere were worth the cost.  Open to suggestions, though.

**Comment by jumblerg** [comment]
i think it would be preferable, in the absence of a better solution, to require the attributes to be imported explicitly.  i'd rather do `(a/div ...)` or `:refer [div]' than encounter the frustration of trying to figure out why my custom "header" element, or the "col" symbol in my let lock, is producing bizarre output when the page renders.

additionally, as components are built up, there's less reliance on these primitive elements; custom components that are used with increased frequency still have to follow the namespacing and scope resolution conventions of the language.  for the sake of consistency, it seems the native elements should as well.
